website/search/elastic_search.py

```diff
@@ -454,6 +454,10 @@ def update_file(file_, index=None, delete=False):
     node_url = '/{node_id}/'.format(node_id=file_.node._id)
 
     parent_url = '/{}/'.format(file_.node.parent_node._id) if file_.node.parent_node else None,
+    parent_title = file_.node.parent_node.title if file_.node.parent_node else None,
+    if parent_title:
+        parent_title = parent_title if file_.node.parent_node.is_public else '-- private project --'
```

Comment:
The text to display here should be handled on the frontend. If the parent title should not be shown, better to pass `null` and have the UI display the proper text.